crates/re_space_view_spatial/src/parts/points2d.rs

```diff
@@ -172,7 +169,15 @@ impl Points2DPart {
 
 impl ViewPartSystem for Points2DPart {
     fn archetype(&self) -> ArchetypeDefinition {
-        Points2D::all_components().try_into().unwrap()
+        vec1::vec1![
```

Comment:
that part we could have kept, no?

Reply:
oh yeah, I guess so, but not going to lose sleep over it